Framework/Muon/test/MuonPairingAsymmetryTest.h

```diff
@@ -360,29 +348,29 @@ class MuonPairingAsymmetryTest : public CxxTest::TestSuite {
   // Correct Output : Single Period
   // --------------------------------------------------------------------------
 
-  void
-  test_that_single_period_data_combines_detectors_correctly_for_manually_specified_detectors() {
-    // 4 spectra per period, 10 bins
-    auto ws = createMultiPeriodAsymmetryData(1, 4, 10, "pairWS");
-    std::vector<int> group1 = {1, 2};
-    std::vector<int> group2 = {3, 4};
+  // void
+  // test_that_single_period_data_combines_detectors_correctly_for_manually_specified_detectors() {
+  //   // 4 spectra per period, 10 bins
+  //   auto ws = createMultiPeriodAsymmetryData(1, 4, 10, "pairWS");
+  //   std::vector<int> group1 = {1, 2};
+  //   std::vector<int> group2 = {3, 4};
```

Comment:
why is this commented out?

Reply:
I don't think it is in the latest commit ?